client/src/components/ProfileBox/index.js

```diff
@@ -33,18 +33,21 @@ const ProfileBox = (props) => {
             </div>
           </div>

```

Comment:
Didn't change any functionality. Just moved elements around. @TimPrimmer lmk if you have any concerns with this.